Yank/restraints.py

```diff
@@ -714,22 +787,24 @@ def _create_restraint_force(self, particle1, particle2):
         parameter_names, parameter_values = zip(*self._bond_parameters.items())
 
         # Create bond force and lambda_restraints parameter to control it.
-        force = openmm.CustomBondForce('lambda_restraints * ' + self._energy_function)
-        force.addGlobalParameter('lambda_restraints', 1.0)
+        if len(particles1) == 1 and len(particles2) == 1:
+            # CustomCentroidBondForce works only on 64bit platforms. When the
```

Comment:
I didn't find it in the docs, but it's in the OpenCL platform code ([here](https://github.com/pandegroup/openmm/blob/master/platforms/opencl/src/OpenCLKernels.cpp#L5054)).